platform/default/mbgl/storage/offline.cpp

```diff
@@ -120,6 +151,23 @@ std::string encodeOfflineRegionDefinition(const OfflineRegionDefinition& region)
             [&] (const OfflineGeometryRegionDefinition& _region) {
                 doc.AddMember("geometry", mapbox::geojson::convert(_region.geometry, doc.GetAllocator()), doc.GetAllocator());
 
+            },
+            [&] (const OfflineTileListRegionDefinition& _region) {
+                rapidjson::GenericValue<rapidjson::UTF8<>, rapidjson::CrtAllocator> bounds(rapidjson::kArrayType);
+                bounds.PushBack(_region.bounds.south(), doc.GetAllocator());
+                bounds.PushBack(_region.bounds.west(), doc.GetAllocator());
+                bounds.PushBack(_region.bounds.north(), doc.GetAllocator());
+                bounds.PushBack(_region.bounds.east(), doc.GetAllocator());
+                doc.AddMember("bounds", bounds, doc.GetAllocator());
+                rapidjson::GenericValue<rapidjson::UTF8<>, rapidjson::CrtAllocator> tileList(rapidjson::kArrayType);
+                for ( auto &i : _region.tileList ) {
```

Comment:
maybe choose a variable besides "i" for iteration, which usually implies a simple int counter.